fhir-search/src/main/java/com/ibm/fhir/search/util/SearchUtil.java

```diff
@@ -687,7 +752,7 @@ public static FHIRSearchContext parseQueryParameters(Class<?> resourceType,
                 throw SearchExceptionUtil.buildNewParseParameterException(name, e);
             }
         } // end for
-
+        
```

Comment:
extra spacing

Reply:
en.. this is interesting, I think all the extra spaces are already removed by eclipsse save action. ah. seems not for this workspace. will remove

Reply:
done